node-geoip2.js

```diff
@@ -1,6 +1,7 @@
 
 var mmdb = require('./lib/node_mmdb.node');
 var path = require('path');
+var exists = require('exists-sync');
```

Comment:
Since `fs.existsSync` is deprecated, I used this microlibrary that accomplishes the same thing in a supported fashion.